demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -900,11 +900,16 @@ def is_valid_fetch(self) -> bool:
         """
         fetch_params_exist = True
         if self.current_file.get('script', {}).get('isfetch') is True:
+
+            # get the iten marketplaces to decide which are the required params
+            marketplaces = get_item_marketplaces(item_path=self.file_path, item_data=self.current_file)
```

Comment:
wouldn't 'get_item_marketplaces' always return a value?

Reply:
@reutshal currently there is one case of that (may be it's a mistake)
see `tools.get_mp_types_from_metadata_by_item`